soapboxpb/environment.proto

```diff
@@ -0,0 +1,71 @@
+syntax = "proto3";
+package soapbox;
+option go_package = "soapboxpb";
+
+import "application.proto";
+
+service Environments {
+    rpc ListEnvironments (ListEnvironmentRequest) returns (ListEnvironmentResponse) {}
+    rpc GetEnvironment (GetEnvironmentRequest) returns (GetEnvironmentResponse) {}
+    rpc CreateEnvironment (CreateEnvironmentRequest) returns (CreateEnvironmentResponse) {}
+    rpc DestroyEnvironment (DestroyEnvironmentRequest) returns (DestroyEnvironmentResponse) {}
+    rpc CopyEnvironment (CopyEnvironmentRequest) returns (CopyEnvironmentResponse) {}
+}
+
+message ListEnvironmentRequest {}
+
+message EnvironmentSummary {
+    int32 id = 1;
+    string name = 2;
+}
+
+message ListEnvironmentResponse {
+    repeated EnvironmentSummary environments = 1;
+}
+
+message GetEnvironmentRequest {
+    int32 id = 1;
+}
+
+message GetEnvironmentResponse {
+    Environment env = 1;
+}
+
+message Environment {
+    int32 id = 1;
+    string name = 2;
+    string slug = 3;
+    bool productionFlag = 4;
+    repeated EnvironmentVariable vars = 5;
+}
+
+message EnvironmentVariable {
+    int32 id = 1;
+    string key = 2;
+    string value = 3;
+    string type = 4;
+    string rate = 5;
+}
+
+message CreateEnvironmentRequest {
```

Comment:
The CreateEnvironmentRequest needs the environment variables too, so we should include a `repeated EnvironmentVariable vars` field here.